doc/source/reference/random/index.rst

```diff
@@ -0,0 +1,219 @@
+that is nearly identical to :class:`~.RandomState`. The canonical
+method to initialize a generator passes a basic RNG -- `~mt19937.MT19937`, the
+underlying RNG in NumPy  -- as the sole argument. Note that the basic RNG must
+be instantized.
+
+.. code-block:: python
+
+  from numpy.random import RandomGenerator, MT19937
+  rg = RandomGenerator(MT19937())
+  rg.random_sample()
+
+Seed information is directly passed to the basic RNG.
+
+.. code-block:: python
+
+  rg = RandomGenerator(MT19937(12345))
+  rg.random_sample()
+
+A shorthand method is also available which uses the `~mt19937.MT19937.
+generator` property from a basic RNG to access an embedded random generator.
+
+.. code-block:: python
+
+  rg = MT19937(12345).generator
+  rg.random_sample()
+
+What's New or Different
+~~~~~~~~~~~~~~~~~~~~~~~
+.. warning::
+
+  The Box-Muller method used to produce NumPy's normals is no longer available
+  in `~.RandomGenerator`.  It is not possible to reproduce the exact random
+  values using ``RandomGenerator`` for the normal distribution or any other
+  distribution that relies on the normal such as the `numpy.random.gamma` or
+  `numpy.random.standard_t`. If you require bitwise backward compatible
+  streams, use `~.mtrand.RandomState`.
+
+* The normal, exponential and gamma generators use 256-step Ziggurat
+  methods which are 2-10 times faster than NumPy's Box-Muller or inverse CDF
+  implementations.
+* Optional ``dtype`` argument that accepts ``np.float32`` or ``np.float64``
+  to produce either single or double prevision uniform random variables for
+  select distributions
+* Optional ``out`` argument that allows existing arrays to be filled for
+  select distributions
+* `~entropy.random_entropy` provides access to the system
+  source of randomness that is used in cryptographic applications (e.g.,
+  ``/dev/urandom`` on Unix).
+* All basic random generators functions can produce doubles, uint64s and
+  uint32s via CTypes (`~xoroshiro128.Xoroshiro128.ctypes`)
+  and CFFI (:meth:`~xoroshiro128.Xoroshiro128.cffi`).
+  This allows these basic RNGs to be used in numba.
+* The basic random number generators can be used in downstream projects via
+  :ref:`Cython <randomgen_cython>`.
+* Support for Lemire’s method [Lemire]_ of generating uniform integers on an
+  arbitrary interval by setting ``use_masked=True`` in
+  `~RandomGenerator.randint`.
+
+See :ref:`new-or-different` for a complete list of improvements and
+differences.
+
+Parallel Generation
+~~~~~~~~~~~~~~~~~~~
+
+The included generators can be used in parallel, distributed applications in
+one of two ways:
+
+* :ref:`independent-streams`
+* :ref:`jump-and-advance`
+
+Supported Generators
+--------------------
+The main innovation is the inclusion of a number of alternative pseudo-random number
+generators, 'in addition' to the standard PRNG in NumPy.  The included PRNGs are:
+
+* MT19937 - The standard NumPy generator.  Produces identical results to NumPy
```

Comment:
"produces identical results to NumPy" is a little confusing now that this is part of numpy :)